tools/ebpf/Dockerfiles/Dockerfile-process-agent-dev

```diff
@@ -0,0 +1,10 @@
+FROM datadog/agent:6.15.1
```

Comment:
you might wanna default to `latest` or add a `--build-arg` for the version so that we don't have to update this

Reply:
Good idea. Is there a nice way to do templating we already use to allow `build-arg` to be customizable?

Reply:
I went with `strings.Template`

https://docs.python.org/3/library/string.html

Reply:
ah yeah this works, but there's also the `--build-arg` parameter to the docker build command (cf https://docs.docker.com/engine/reference/commandline/build/)